scripts/msmarco-passage/create_faiss_index.py

```diff
@@ -0,0 +1,34 @@
+import argparse
+import os
+
+import faiss
+import numpy as np
+import pandas as pd
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--embedding", type=str, help='encoder name or path', required=True)
+    parser.add_argument("--index-type", type=str, help='doc file to get encoded.', required=True)
+    parser.add_argument("--index-dir", type=str, help='doc type, [passage, query]', required=True)
```

Comment:
Just `--index` for index location? To be consistent with sparse indexes and anserini.

Reply:
sure